### DISCUSSION TITLE
OpenMP Support

### DISCUSSION THREAD

**Discussion by arnoldjames98** [exaloop/codon#310] [General]
Is it possible to call functions like `omp_get_num_threads()` or include complier directives like `#pragma omp critical` within Python scripts in complied with Codon? If so, are there any examples available online?

**Comment by arshajii** [comment]
Hi @arnoldjames98 -- yes this is all supported, please see https://docs.exaloop.io/codon/advanced/parallel#openmp-constructs for more info/examples.
